packages/ui/src/2_molecules/Header/Header.module.css

```diff
@@ -0,0 +1,19 @@
+.header {
+  @apply bg-gray-90 h-16 w-full flex align-middle justify-between px-11;
```

Comment:
@tiltom since we have a margin in the logo with 44px from both sides, I believe it would be more sensible to leave it 44px from the right side of the header